tests/ethpm/test_package_init_from_registry.py

```diff
@@ -39,20 +29,22 @@ def w3_with_registry(w3):
     )
     # register package on deployed registry
     registry.functions.registerPackage(
-        b'owned',  # pkg-name
+        b'safe-math-lib',  # pkg-name
         '1.0.0',  # version
-        'ipfs://QmbeVyFLSuEUxiXKwSsEjef6icpdTdA4kGG9BcrJXKNKUW'  # content-addressed URI
+        'ipfs://QmWgvM8yXGyHoGWqLFXvareJsoCZVsdrpKNCLMun3RaSJm'  # content-addressed URI
     ).transact()
     w3.eth.waitForTransactionReceipt(tx_hash)
     return w3, address, registry
 
 
 def test_package_init_from_registry(w3_with_registry):
```

Comment:
Same here, not sure how to avoid the http req - I'd like to use a real `valid_registry_uri` that is run through the validation utils - but also points to a valid IPFS content-addressed uri.

Reply:
I'd suggest using a dummy backend which stores a local mapping of `URI -> content` and blows up if you ask it for anything it doesn't know about.

Reply:
Thoughts on using the `requests-mock` library, like I did in a [subsequent pr](https://github.com/ethpm/py-ethpm/pull/60/files#diff-d76056684bd2b90a26b7829883ac9614R30). It's not ideal right now since it mocks ANY http request, I'm working on how to only mock a specific request and blow up for others (a little tricky since the configuration I'm using isn't the default one in the docs).